components/camel-tracing/src/main/java/org/apache/camel/tracing/ActiveSpanManager.java

```diff
@@ -56,6 +56,13 @@ public static SpanAdapter getSpan(Exchange exchange) {
      * @param span     The span
      */
     public static void activate(Exchange exchange, SpanAdapter span) {
+        if (exchange.getProperty(ExchangePropertyKey.CLOSE_CLIENT_SCOPE, Boolean.FALSE, Boolean.class)) {
```

Comment:
We can use removeProperty to not keep it around anymore as we have ended the scope